modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/OIDCUserInfoProcessor.java

```diff
@@ -830,7 +830,7 @@ private void _saveOpenIdConnectUser(
 		}
 
 		_openIdConnectUserLocalService.addOpenIdConnectUser(
-			user.getCompanyId(), user.getUserId(), issuer, subject);
+			user.getUserId(), issuer, subject);
```

Comment:
remove this change from this commit and put together with the one that adds this line.